### PR TITLE
Fix tooltip arrow positioning

### PR DESCRIPTION
**Pull request by ClemDelp**
Two arrow positioning fixes on small screen:
- when position = bottom / top & arrow right
image

- when position = bottom / top & arrow = center the position is not really center:
image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,6 +3,9 @@ import PropTypes from 'prop-types'
 import ReactDOM, {unstable_renderSubtreeIntoContainer as renderSubtreeIntoContainer} from 'react-dom'
 import assign from 'object-assign'
 
+let FG_SIZE = 8
+let BG_SIZE = 9
```
Comment (Redmega) [inline src/index.js]: If these are not modified they should be `const`.

Comment (ClemDelp) [reply src/index.js]: Oups yes you right, I fixed that 👍

File: src/index.js [hunk 2]
```diff
@@ -226,10 +223,21 @@ class Card extends React.Component {
...
+        const parent = this.props.parentEl
+        if (parent) {          
+          const tooltipWidth = this.state.width
+          let bgStyleRight = arrowStyle.bgStyle.right
+          // For arrow = center
+          if (!bgStyleRight) {
+            bgStyleRight = (tooltipWidth / 2) - BG_SIZE
+          }
+          const newBgRight = Math.round(bgStyleRight - style.left + this.margin)
+          arrowStyle = Object.assign({}, arrowStyle, {
```
Comment (romainberger) [inline src/index.js]: could use `assign` from `object-assign` for consistency

Comment (ClemDelp) [reply src/index.js]: Done 👍

Comment (romainberger) [reply src/index.js]: Thanks!

**Comment by Redmega** [comment]
:tada: I'll work on getting this merged and released tomorrow if nobody has any objections.

**Pull request merged**